### DISCUSSION TITLE
Memory Pressure Experiment

### DISCUSSION THREAD

**Discussion by rimbesat** [DataDog/chaos-controller#938] [Ideas, user requests and proposals]
Hi team,

I was wondering if you have any plans or thoughts on including memory pressure experiments?

**Comment by ptnapoleon** [comment]
Hi @rimbesat, could you expand on what you're imagining? We haven't found a situation where we can provide memory pressure in a useful way, that doesn't just result in a container failure: from OOM killer, or kubernetes pod eviction.

**Comment by aymericDD** [comment]
Hi @rimbesat 
The memory pressure is now supported https://github.com/DataDog/chaos-controller/blob/main/docs/memory_disruption.md 🥳
